app/DoctrineMigrations/Version20160410190541.php

```diff
@@ -33,8 +33,9 @@ public function up(Schema $schema)
 
         $this->skipIf($entryTable->hasColumn('uuid'), 'It seems that you already played this migration.');
 
-        $entryTable->addColumn('uuid', 'guid', [
+        $entryTable->addColumn('uuid', 'string', [
             'notnull' => false,
+            'length' => 23,
```

Comment:
Yep but we don't generate a UUID, because we are using `uniqid()` which doesn't generate an UUID.